web/sites/default/config/field.field.paragraph.form_app_button.field_form_button.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 43475402-0ed1-44ff-bbb1-f3314bbea136
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_form_button
+    - paragraphs.paragraphs_type.form_app_button
+  module:
+    - link
+id: paragraph.form_app_button.field_form_button
+field_name: field_form_button
+entity_type: paragraph
+bundle: form_app_button
+label: 'Form link'
+description: ''
+required: false
```

Comment:
Please make the field required. Paragraphs will only require them to fill it out if a Form/App Button is added to the step, but if it exists, it should be required.